www/cgi-bin/minerinfo

```diff
@@ -5,13 +5,20 @@
 # Rev 0004 :Average Hash rate for zm and ethminer
 
 export PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+export NVOC=/home/m1
```

Comment:
Cant it gets NVOC path from .profile ?

Reply:
.profile is only loaded by shells, apache web server does not  use .profile

People may prefer to export that variable from different point, I think simply seeking it from .profile is not our best option

Reply:
So NVOC path should be set in 2unix, .profile and minerinfo ?

Reply:
Yup, but since minerinfo already searches for a miner by filtering running processes I think it could be ok if we extract that path from the first part of the miner path

Reply:
Agree, So we can extract it from 2unix?

Reply:
Some thing like :
grep "export" /home/m1/2unix | awk '{print $2}' |sed 's/NVOC=//'

Reply:
You cannot, how do you know where 2unix is?

Reply:
Now evaluating this: 
ps ax | grep -i 2unix | grep -o '[^_[:space:]]*$'

Reply:
Dont we need to copy 2unix from NVOC directory to /home/m1/ for it to start at boot ?

Reply:
Nope, just change or create a new gnome-terminal profile with the updated path, that's how it works now. I'm know evaluating different alternatives, I'll let you know, maybe I could also avoid exporting it in 2unix.

Reply:
At the end choosing the nvOC version to run should only consists in selecting the default gnome-terminal profile

Reply:
What I say is that we can switch to different versions or upgrade with least changes, so that users can only change 1 file and get all up and running.

Reply:
What if we have a /home/m1/nvoc_location file with just a path inside and all the rest read from it ?
Either its /home/m1/ or /home/m1/Downloads/nvoc/19-2.1 or ...